src/config/types.ts

```diff
@@ -0,0 +1,37 @@
+export interface AppConfig {
```

Comment:
do we need this type? Seems redundant to the actual config. Can we not export as const? and derive the type from it?

```
export const config = {} as const;
export type Config = typeof config;
```

Reply:
The main goal of this interface is to make sure all modes have the same keys. I can derive it but I see two issues : 
- One of the config must be the reference (ethereum/production for example).
- `as const` will make string as string literal. So we would have to hard type them in the reference object to guide typescript.
- It makes things a little bit less explicit. Since this config folder is dedicated to external team, I think it's better to explicitly create a readable interface they can look at.